cts/cts-scheduler.in

```diff
@@ -1578,7 +1584,9 @@ class CtsScheduler(object):
         else:
             rc = self.run_one(self.args.run, "Single shot", self.single_test_args)
             self.failed_file.close()
-            cat(self.failed_filename)
+            if self.num_failed > 0:
+              print("\nFailures:\nThese have also been written to: " + self.failed_filename + "\n")
+              cat(self.failed_filename)
```

Comment:
and remove the directory if no failure here (which we weren't doing before, but would be a good idea)